### PR TITLE
Telemetry logging

### PR DESCRIPTION
**Pull request by k88hudson**
I'm also gonna write up a script for checking each event

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/activity-streams/587)

### CODE REVIEW

**Comment by k88hudson** [comment]
R? @rlr

File: data_events.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+This is an overview of the different kinds of data we collect in Activity Stream experiment.
+
+## User event pings
+
+These pings are captured when a user **performs some kind of interaction** in the add-on.
+
+### Basic shape
+
+The variable properties in a use event ping include some variable feels which indicate the location and action of the event, as well as some basic metadata.
```
Comment (rlr) [inline data_events.md]: `s/feels/fields` I think?

File: data_events.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  // This is where the interaction occurred
+  "page": "[NEW_TAB | TIMELINE_ALL | TIMELINE_BOOKMARKS]",
+
+  // Optional field indicating the UI component type
+  "source": ["TOP_SITES" | "FEATURED" | "ACTIVITY_FEED"],
+
+  // Optional field if there is more than one of a component type on a page.
+  // It is zero-indexed.
+  // For example, clicking the second Highlight would result in an action_position of 1
+  "action_position": 1
```
Comment (rlr) [inline data_events.md]: missing `,`

**Comment by rlr** [comment]
LGTM! R+ w/ typo fixes

File: data_events.md (hunk 1 above)
Comment (k88hudson) [inline data_events.md]: hahahahaha

Comment (pdehaan) [inline data_events.md]: +1 to leaving the typo as-is, because a self-aware add-on is amusing.

File: data_events.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+This is an overview of the different kinds of data we collect in Activity Stream experiment.
+
+## User event pings
+
+These pings are captured when a user **performs some kind of interaction** in the add-on.
+
+### Basic shape
+
+A user event ping includes some basic metadata (tab id, addon version, etc.) as well as variable fields which indicate the location and action of the event.
```
Comment (pdehaan) [inline data_events.md]: "addon" or "add-on"? (we hyphenated on L7 above)

**Pull request merged**